packages/curve-ui-kit/src/widgets/Header/HeaderLogo.tsx

```diff
@@ -36,17 +37,17 @@ export const HeaderLogo = ({ currentMenu, isLite, sx }: HeaderLogoProps) => (
       >
         {APP_NAMES[currentMenu]}
       </Typography>
-      {currentMenu === 'dex' ? (
-        isLite && (
-          <Typography variant="bodyXsBold" color="textTertiary">
-            Lite
-          </Typography>
-        )
-      ) : (
-        <Typography variant="bodyXsRegular" color="textTertiary">
-          powered by Curve
-        </Typography>
-      )}
+      {currentMenu === 'dex'
+        ? isLite && (
+            <Typography variant="bodyXsBold" color="textTertiary">
+              Lite
+            </Typography>
+          )
+        : currentMenu !== 'analytics' && (
```

Comment:
Again, hardcoding analytics isn't really maintainable
```suggestion
        : APP_NAMES[app] !== 'Curve' && (
```